themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -86,7 +86,11 @@
     <body>
 
         <script>
-            var keycloak = Keycloak('${authUrl}/realms/${realm.name}/account/keycloak.json');
+            var keycloak = Keycloak({
```

Comment:
A question here.  Long ago we decided to download keycloak.json.  Here, the config is hard-coded.  Is this what we want long term?

Reply:
Yes, there's just no need to have that extra call